src/components/Tooltip/Tooltip.tsx

```diff
@@ -15,7 +15,9 @@ type TooltipProps = {
   /**
    * Behavior of the tooltip transition, defaults to an opacity "fade".
    * Animation guidelines are provided in https://atomiks.github.io/tippyjs/v5/animations/.
-   * A false value will disable animations.
+   *
+   * Set animation={false} to disable animations **in tests only**.
+   * For some reason this also hides the Tooltip visually.
```

Comment:
Is this still accurate? Since you're changing the `duration` in this PR.

Reply:
hmm there was one tooltip test (the escape key one) passing `animation={false}`. lemme see if `duration` works instead

Reply:
duration works! will update comment